server/lib/service/dv360.dart

```diff
@@ -0,0 +1,27 @@
+import 'dart:async';
+
+import 'package:fixnum/fixnum.dart';
+import 'package:googleapis/displayvideo/v1.dart';
+import 'package:http/http.dart';
+
+/// A class that wraps around Display & Video 360.
+class DisplayVideo360 {
+  /// The DV360 API.
+  final DisplayvideoApi _api;
```

Comment:
capital letter?

Reply:
This is the googleapis' [DV360 class](https://pub.dev/documentation/googleapis/latest/googleapis.displayvideo.v1/DisplayvideoApi-class.html). I'm not sure why they didn't capitalize it.